### PR TITLE
adding some lil styles

### PR DESCRIPTION
**Pull request by mfrachet**
screenshot 2018-12-28 at 13 43 13

NB: if you see insanity, that's because I m a CSS shit \o/

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import styled from "styled-components";
+import "./index.css";
```
Comment (Swiip) [inline src/index.js:4]: 😱

File: src/components/form-content.js [hunk 2]
```diff
@@ -94,29 +132,56 @@ const FormContent = ({ history }) => {
...
-          <Legend>First oponent</Legend>
+          <Legend>First fighter</Legend>
           <PokemonListContainer>
-          {result.map(pokemon => (
-            <Fragment key={pokemon.name}>
-              <input type="radio" onChange={handleFirst} id={`oponent1-${pokemon.name}`} value={pokemon.name} name="oponent1" />
-              <label htmlFor={`oponent1-${pokemon.name}`}>{pokemon.name}</label>
-            </Fragment>
-          ))}
+            {result.map((pokemon, index) => (
```
Comment (Swiip) [inline src/components/form-content.js:137]: This component should be split in severals

File: src/components/form-content.js [hunk 3]
```diff
@@ -94,29 +132,56 @@ const FormContent = ({ history }) => {
...
+                <input
+                  type="radio"
+                  onChange={handleSecond}
+                  id={`oponent2-${pokemon.name}`}
+                  value={pokemon.name}
+                  name="oponent2"
+                />
+                <label htmlFor={`oponent2-${pokemon.name}`}>
+                  <Img
+                    src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${index +
```
Comment (Swiip) [inline src/components/form-content.js:171]: This will have to be passed in the scrapper

**Review by Swiip — APPROVED** [review]
Totally in love with the design :)

Approve despite the comments as we have to go forward

File: src/components/form-content.js (hunk 2 above)
Comment (mfrachet) [reply src/components/form-content.js:137]: absolutely!

**Pull request merged**